app.rb

```diff
@@ -1,9 +1,34 @@
 require 'sinatra/base'
+require './lib/peeps'
+require 'time'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+  get '/' do
+    @peeps = Peeps.all
+    p @peeps
+    erb :'index' 
+  end
+
+  get '/new' do
+    erb :'new'
+  end
+
+  post '/' do
+    message = params['message']
+    p params['message']
+    time = Time.now.httpdate
+    connection = PG.connect(dbname: 'chitter')
```

Comment:
Yes, that would be a good next step. It would allow you to test this code in a unit test just like you tested the `Peep.all` method in `peeps_spec.rb`. 
It would also make the separation between Controller and Model that I talked about above clearer.